pkgs/by-name/op/opener/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "opener";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "superbrothers";
+    repo = "opener";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-pp2jn4J6gYyZky5rVBaai9sVVqvPgM+fGryseM58WEI=";
+  };
+
+  vendorHash = "sha256-xwMRWEwrG12QevVVTMC9OTdjIBoxR1AHkoa6OErPpF4=";
+
+  meta = {
+    description = "Open URL in your local web browser from the SSH-connected remote environment";
+    homepage = "https://github.com/superbrothers/opener";
+    changelog = "https://github.com/superbrothers/opener/releases/tag/v${version}";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.all ];
```

Comment:
don't shift the burden onto all of us, just add yourself (or find someone who maintains it)

Reply:
Honest mistake sorry

Reply:
happens to the best of us ;)